data/assets/nightsky/ecliptic_band.asset

```diff
@@ -22,7 +22,7 @@ local EclipticLine = {
   Transform = {
     Scale = {
       Type = "StaticScale",
-      Scale = 4.28601E17
+      Scale = 9.46377307652E17
```

Comment:
What's the reason to shrink this radius by 2?

Reply:
Cleanup/simplicity, I assume. Note that the `Radii = { 0.5, 0.5 }` below is removed and it now uses the default value

Reply:
And it's increased, not shrunk :) 

The sphere below has also been increased in the same way, likely to be made to have the same size as this circle